### PR TITLE
Feature/svg icons

### PR DESCRIPTION
**Pull request by dietertaotesting**
[Download these files, they will help you to validate the PR](https://github.com/oat-sa/tao-core/files/242004/test-instructions.zip)

### CODE REVIEW

File: views/js/lib/polyfill/svg4everybody/svg4everybody.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                    target || (target = xhr._cachedTarget[item.id] = cachedDocument.getElementById(item.id)), 
+                    // embed the target into the svg
+                    embed(item.svg, target);
+                });
+            }
+        }, // test the ready state change immediately
+        xhr.onreadystatechange();
+    }
+    function svg4everybody(rawopts) {
+        console.log('svg4everybody called');
```
Comment (krampstudio) [inline views/js/lib/polyfill/svg4everybody/svg4everybody.js]: Could you please remove the `console.log`

File: models/classes/menu/Tree.php [hunk 2]
```diff
@@ -28,8 +28,13 @@ class Tree  implements PhpSerializable
...
     private $data = array();
-    
-    public static function fromSimpleXMLElement(\SimpleXMLElement $node) {
+
+    /**
+     * @param \SimpleXMLElement $node
+     * @param $structureExtensionId
+     * @return static
+     */
+    public static function fromSimpleXMLElement(\SimpleXMLElement $node, $structureExtensionId) {
```
Comment (krampstudio) [inline models/classes/menu/Tree.php]: are you planning to use `$structureExtensionId` here ?

File: views/js/ui.js [hunk 3]
```diff
@@ -15,6 +15,23 @@ define([
     'ui/groupvalidator'
 ], function(toggler, disabler, adder, deleter, incrementer, inplacer, btngrouper, durationer, selecter, modal, tooltipster, form) {
     'use strict';
+
+    function initCrossBrowserSvg() {
```
Comment (krampstudio) [inline views/js/ui.js]: Could you add a small comment about why you need to load this library, etc.

File: views/scss/inc/_header-footer.scss [hunk 4]
```diff
@@ -97,6 +97,34 @@ $navHeight: 64;
                     position: relative;
                     top: 2px;
                 }
+                svg {
```
Comment (krampstudio) [inline views/scss/inc/_header-footer.scss]: Shouldn't we have a function or a mixin to ensure the structure is used correctly ?

**Comment by krampstudio** [comment]
I'm agree on the principle, the js implementation looks correct. 
@jbout could you please have a just a quick look at the extension/structure part.

**Comment by jbout** [comment]
structure makes sense, but test-cases need to be updated

**Comment by no-chris** [comment]
@krampstudio you can review the changes we discussed

**Comment by krampstudio** [comment]
Ready to be merged from my POV, but I don't have the time right now. Who can take care of the merge + basic testing ?

**Comment by no-chris** [comment]
I will merge this asap

**Pull request merged**